src/test_utils.rs

```diff
@@ -76,6 +104,109 @@ impl TestContext {
             .await
             .unwrap();
     }
+
+    /// Retrieve a sent message from the jobs table.
+    ///
+    /// This retrieves and removes a message which has been scheduled to send from the jobs
+    /// table.  If there are multiple messages scheduled to send you may get any of them.
```

Comment:
Maybe use `ORDER BY` to make sure messages are retrieved in deterministic way? Delta Chat relies on messages being processed in UID order.